packages/ceramic-cli/src/ceramic-cli-utils.ts

```diff
@@ -0,0 +1,312 @@
+import CeramicClient from "@ceramicnetwork/ceramic-http-client"
+import { CeramicApi, DoctypeUtils } from "@ceramicnetwork/ceramic-common"
+
+const PREFIX_REGEX = /^ceramic:\/\/|^\/ceramic\//
+
+/**
+ * Ceramic CLI utility functions
+ */
+export default class CeramicCliUtils {
+
```

Comment:
Nice that you pulled these out. Cli class is very clean now :)

Reply:
🚀